src/lib/refreshTokenlib.js

```diff
@@ -0,0 +1,10 @@
+const bcrypt = require('bcrypt')
+
+const verifyRefreshToken = async (oldRefreshToken, refreshToken) => {
+	const isSimilar = await bcrypt.compare(oldRefreshToken, refreshToken)
```

Comment:
we only use `bcrypt` for the password because we encrypt it before entering it into the database (if you check the `createNewUser` implementation)  so the `compare` is to try to see if the provided password matches the encrypted one

Reply:
oh okay, that makes so much sense, thank you